config/_default/menus/menus.en.yaml

```diff
@@ -2337,7 +2342,8 @@ main:
     parent: tracing_troubleshooting
     weight: 1306
   - name: Correlated Logs
-    url: tracing/troubleshooting/correlated-logs-not-showing-up-in-the-trace-id-panel
+    url: >-
```

Comment:
What does this `>-` do? Is that intentional? The link still appears to work correctly in the preview, but I'm just curious.